java.lang.NullPointerException: Cannot invoke "String.toUpperCase()"

Hi folks. I see an error with OpenJDK 16 and using synthea-international example with synthea. I speculate that switching Java versions will sort this out and that this is more about synthea than the international repo (hence posting here). I'm curious about the recommended Java distributions and versions if that is indeed the issue.

Here's the error and steps to reproduce (on macOS):
```
> Task :compileJava
/Users/richard/src/github.com/synthetichealth/synthea/src/main/java/org/mitre/synthea/world/geography/Location.java:159: warning: [removal] Double(double) in Double has been deprecated and marked for removal
          Double sum = averages.getOrDefault(determinant, new Double(0));
                                                          ^
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 warning

> Task :run FAILED
WARNING: unable to load SDoH csv: null
java.lang.NullPointerException
        at java.base/java.util.Objects.requireNonNull(Objects.java:208)
        at java.base/java.lang.ClassLoader.getResource(ClassLoader.java:1393)
        at com.google.common.io.Resources.getResource(Resources.java:194)
        at org.mitre.synthea.helpers.Utilities.readResource(Utilities.java:348)
        at org.mitre.synthea.world.geography.Location.<init>(Location.java:127)
        at org.mitre.synthea.engine.Generator.init(Generator.java:206)
        at org.mitre.synthea.engine.Generator.<init>(Generator.java:189)
        at org.mitre.synthea.engine.Generator.<init>(Generator.java:174)
        at App.main(App.java:219)
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "String.toUpperCase()" because the return value of "org.mitre.synthea.world.geography.Location.getAbbreviation(String)" is null
        at org.mitre.synthea.world.agents.Payer.loadPayers(Payer.java:207)
        at org.mitre.synthea.world.agents.Payer.loadPayers(Payer.java:177)
        at org.mitre.synthea.engine.Generator.init(Generator.java:247)
        at org.mitre.synthea.engine.Generator.<init>(Generator.java:189)
        at org.mitre.synthea.engine.Generator.<init>(Generator.java:174)
        at App.main(App.java:219)

FAILURE: Build failed with an exception.
```

```
$ java -version
openjdk version "16.0.1" 2021-04-20
OpenJDK Runtime Environment Homebrew (build 16.0.1+0)
OpenJDK 64-Bit Server VM Homebrew (build 16.0.1+0, mixed mode, sharing)
```

To try to reproduce:
```
git clone https://github.com/synthetichealth/synthea
git clone https://github.com/synthetichealth/synthea-international
cd synthea-international
cp -R example/* ../synthea
cd ../synthea
./run_synthea -p 5
```

--- a/src/main/java/org/mitre/synthea/world/geography/Location.java
+++ b/src/main/java/org/mitre/synthea/world/geography/Location.java
@@ -123,7 +123,8 @@ public class Location implements Serializable {
 
     socialDeterminantsOfHealth = new HashMap<String, Map<String, Double>>();
     try {
-      filename = Config.get("generate.geography.sdoh.default_file");
+      filename = Config.get("generate.geography.sdoh.default_file",
+        "geography/sdoh.csv");
       String csv = Utilities.readResource(filename);
       List<? extends Map<String,String>> sdohList = SimpleCSV.parse(csv);
 

